__tests__/organization.test.

```diff
@@ -0,0 +1,127 @@
+const { MongoMemoryServer } = require('mongodb-memory-server'); 
+const mongod = new MongoMemoryServer(); 
+const mongoose = require('mongoose');
+const connect = require('../lib/utils/connect.js'); 
+
+const request = require('supertest'); 
+const app = require('../lib/app'); 
+const Organization = require('../lib/models/Organization'); 
+
+describe('organization routes', () => {
+
+  beforeAll(async() => {
+    const uri = await mongod.getUri();
+    return connect(uri);
+  });
+    
+  beforeEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
+  
+  // let poll; 
+  // beforeEach(async() => {
+  //   poll = await poll.create({
+  //     create
+  //   });
+  // }); 
+
+  afterAll(async() => {
+    await mongoose.connection.close();
+    return mongod.stop();
+  });
+
+  it('creates an organization via POST', () => {
+    return request(app)
+      .post('/api/v1/organizations')
+      .send({
+        title: 'Environmental Voter Project 1',
+        description: 'description1',
+        imageUrl: 'image1.com'
+      })
+      .then (res => {
+        expect(res.body).toEqual({
+          _id: expect.anything(),
+          title: 'Environmental Voter Project 1', 
+          description: 'description1', 
+          imageUrl: 'image1.com', 
+          __v: 0
```

Comment:
It looks like you don't need the __v: 0 in your expect block because you've already removed them in your Model.